packages/webview_flutter/lib/src/tizen_webview_controller.dart

```diff
@@ -0,0 +1,390 @@
+      case LoadRequestMethod.post:
+        break;
+    }
+    // The enum comes from a different package, which could get a new value at
+    // any time, so a fallback case is necessary. Since there is no reasonable
+    // default behavior, throw to alert the client that they need an updated
+    // version. This is deliberately outside the switch rather than a `default`
+    // so that the linter will flag the switch as needing an update.
+    // ignore: dead_code
+    throw UnimplementedError(
+        'This version of `TizenWebViewController` currently has no '
+        'implementation for HTTP method ${params.method.serialize()} in '
+        'loadRequest.');
+  }
+
+  @override
+  Future<String?> currentUrl() => _webview.currentUrl();
+
+  @override
+  Future<bool> canGoBack() =>
+      _webview.canGoBack().then((bool? result) => result!);
+
+  @override
+  Future<bool> canGoForward() =>
+      _webview.canGoForward().then((bool? result) => result!);
+
+  @override
+  Future<void> goBack() => _webview.goBack();
+
+  @override
+  Future<void> goForward() => _webview.goForward();
+
+  @override
+  Future<void> reload() => _webview.reload();
+
+  @override
+  Future<void> clearCache() => _webview.clearCache();
+
+  @override
+  Future<void> clearLocalStorage() {
+    throw UnimplementedError(
+        'This version of `TizenWebViewController` currently has no '
+        'implementation.');
+  }
+
+  @override
+  Future<void> setJavaScriptMode(JavaScriptMode javaScriptMode) =>
+      _webview.setJavaScriptMode(javaScriptMode.index);
+
+  @override
+  Future<String?> getTitle() =>
+      _webview.getTitle().then((String? result) => result!);
+
+  @override
+  Future<void> scrollTo(int x, int y) => _webview.scrollTo(x, y);
+
+  @override
+  Future<void> scrollBy(int x, int y) => _webview.scrollBy(x, y);
+
+  @override
+  Future<Offset> getScrollPosition() =>
+      _webview.getScrollPosition().then((Offset? result) => result!);
+
+  @override
+  Future<void> setBackgroundColor(Color color) =>
+      _webview.setBackgroundColor(color);
+
+  @override
+  Future<void> setPlatformNavigationDelegate(
+      covariant TizenNavigationDelegate handler) async {
+    handler.setMethodCallHandler(_webview);
+    handler.setOnLoadRequest(loadRequest);
+    _webview.setHasNavigationDelegate(true);
+  }
+
+  @override
+  Future<void> addJavaScriptChannel(
+    JavaScriptChannelParams javaScriptChannelParams,
+  ) async {
+    _webview.addJavaScriptChannel(javaScriptChannelParams);
+  }
+
+  @override
+  Future<void> runJavaScript(String javaScript) =>
+      _webview.runJavaScript(javaScript);
+
+  @override
+  Future<Object> runJavaScriptReturningResult(String javaScript) => _webview
+      .runJavaScriptReturningResult(javaScript)
+      .then((Object? result) => result!);
+
+  @override
+  Future<void> setUserAgent(String? userAgent) =>
+      _webview.setUserAgent(userAgent);
+}
+
+/// An implementation of [PlatformWebViewWidget] with the WebKit api.
```

Comment:
"WebKit API" is ambiguous since it usually refers to Apple's browser engine.

Reply:
done